src/modules/views/analytics/AnalyticsView.tsx

```diff
@@ -0,0 +1,75 @@
+import { useTranslation } from 'react-i18next';
+
+import { Alert, Box } from '@mui/material';
+
+import { AppAction } from '@graasp/sdk';
+import { Loader } from '@graasp/ui';
+
+import { differenceInSeconds } from 'date-fns';
+import groupBy from 'lodash.groupby';
+import orderBy from 'lodash.orderby';
+
+import { APP_ACTIONS_TYPES } from '@/config/appActionsTypes';
+import { hooks } from '@/config/queryClient';
+import { CodeVersionType } from '@/interfaces/codeVersions';
+
+import { ANALYTICS_VIEW_CY } from '../../../config/selectors';
+import GeneralStatistics from './GeneralStatistics';
+import UsersRunningCodeVersions from './UsersRunningCodeVersions';
+
+const AnalyticsView = (): JSX.Element => {
+  const { data, isLoading } = hooks.useAppActions();
+
+  const { t } = useTranslation();
+
+  if (data) {
+    const actionsOrdersByCreatedDate = orderBy(data, 'createdAt');
+    const actionsByType = groupBy(actionsOrdersByCreatedDate, 'type');
+    const actionsByMember = groupBy(actionsOrdersByCreatedDate, 'member.id');
+
+    const generalStatistic = Object.keys(actionsByMember).map((ele) => {
```

Comment:
If you use `Object.entries(actionsByMember).map(([memberId, memberActions]) => `you would already have `actionsByMember[ele]` as `memberActions`.